css/selectors/column.json

```
@@ -0,0 +1,65 @@
+{
+  "css": {
+    "selectors": {
+      "combinator": {
```

I'm thinking "column" is a more appropriate ident (especially considering the filename).  Can we update that?

```suggestion
      "column": {
```